src/scaffold/zome/integrity.rs

```diff
@@ -56,134 +55,133 @@ pub fn validate_agent_joining(_agent_pub_key: AgentPubKey, _membrane_proof: &Opt
 ///
 /// You can read more about validation here: https://docs.rs/hdi/latest/hdi/index.html#data-validation
 #[hdk_extern]
-pub fn validate(op: Op) -> ExternResult<ValidateCallbackResult> {{
-    match op.flattened::<(), ()>()? {{
-        FlatOp::StoreEntry(store_entry) => match store_entry {{
-            OpEntry::CreateEntry {{
+pub fn validate(op: Op) -> ExternResult<ValidateCallbackResult> {
```

Comment:
Nice! Really appreciated to fix some of the lint issues for generated code too. I'm not sure how practical it is to ensure this at every stage but I would appreciate `cargo clippy` passing on newly scaffolded projects most of the time. At least with the Rust toolchain that's provided in Holonix, nothing we can do about clippy getting new lints over time etc :)

Reply:
> I would appreciate cargo clippy passing on newly scaffolded projects most of the time.

Ahh, I had not considered this too, will fix any issues on the generated code

Reply:
Fixed issues with generated rust code, clippy should now pass for the zome crates. I would also like to consider programatic formatting of the generated rust code to ensure `cargo fmt` also passes, I found [this code snippet](https://github.com/bcmyers/num-format/blob/v0.1.2/num-format-dev/src/rustfmt.rs) that could be useful with that. Will do this in a separate PR though